x.toml

```diff
@@ -157,19 +157,19 @@ members = [
 # Interesting subsets of the workspace, These are used for generating and
 # checking dependency summaries.
 
-[workspace.subsets.lsr]
+[subsets.lsr]
```

Comment:
These really aren't subsets.   They are nodes used to define a subset via a transformation of finding all in workspace dependencies, correct.   The naming sits a little wrong with me and think this needs to be aggressively documented at the various points where this could be misused...   still chewing on this though.

Reply:
Yeah, good point. Maybe it should be `root-members` instead of `members` to signal this.

Reply:
Alright, been chewing on this.   If we want to use the subset terminology we should probably have something like

default-members.subset.lsr to indicate these configurations behave the same as default members would....  

Happy to be convinced otherwise.

Reply:
heh didn't see your comment before I made my second one.... wierd.

Reply:
I've changed it to `root-members` -- my concern with `default-members` is that it's a bit too specific. I think `root-members` is the right level of generality and communicates this information well. (We can also add `members` in the future if we *don't* want to traverse dependencies.